src/seed/seedSuperAdmin.js

```
@@ -1,26 +1,29 @@
-        password,
+        firstName: process.env.MAIL_FIRSTNAME,
+        lastName: process.env.MAIL_LASTNAME,
+        email: process.env.MAIL_USER,
+        password: hashedStudentPassword,
+        lastLoginAs: ADMIN,
         active: true,
         isEmailConfirmed: true
       }
 
-      return await User.create(superAdmin)
+      const createdUser = await User.create(superAdmin);
```

ESLint

[error] Extra semicolon.

(semi)